src/Uno.UI/Helpers/UnoRuntimePlatform.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Uno.UI.Helpers;
+
+internal enum UnoRuntimePlatform
+{
+	Android,
+	iOS,
+	MacCatalyst,
+	MacOSX,
+	WebAssembly,
+	Windows,
+	Skia,
+	SkiaGtk,
+	SkiaWpf,
+	SkiaX11,
+	SkiaFrameBuffer,
+	SkiaMacOS,
```

Comment:
SkiaIslands?

Reply:
In such case I would go with `SkiaWpfIslands` to make it even more specific